myapp/spec/factories/labels.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :label do
+    name { "MyString" }
+    association :user
```

Comment:
You don't have to explicitly use `association`. Instead, you can just `user` like this:

```suggestion
    user
```

This page might be helpful.

https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#associations

Reply:
Noted. Thank you for the material 🙏